config/sync/views.view.gp_practices_proximity.yml

```diff
@@ -71,14 +71,12 @@ display:
             offset_label: Offset
           quantity: 9
       style:
-        type: html_list
+        type: default
         options:
-          row_class: ''
+          grouping: {  }
+          row_class: card
           default_row_class: false
-          uses_fields: true
-          type: ul
-          wrapper_class: item-list
-          class: ''
+          uses_fields: false
```

Comment:
Are you sure this should be false? 
CC @johangant

Reply:
pretty sure! Might be changed later, but for now it needs to match the other view

Reply:
I think it's fine to change the format options to unstyled list. The key thing that needs to be enabled is the 'Force using fields' option as without it, the proximity search query fails.

Reply:
Hard to say whether that option maps to the 'uses_fields' directive in there. The general idea is you want to show Content (rather than individual fields) in a specific view mode, and you want that obscure 'Force fields' option enabled to ensure the query is well-formed.